downstream/modules/platform/proc-configuring-the-metrics-utility.adoc

```diff
@@ -3,23 +3,22 @@
 :_mod-docs-content-type: PROCEDURE
 
 [id="configuring-the-metrics-utility_{context}"]
-= configuring-the-metrics-utility
+= Configuring metrics-utility
 
 
 == On RHEL 
 
-=== Prerequisites:
-
-* An active Ansible Automation Platform subscription
+.Prerequisites:
 
+* An active {PlatformNameShort} subscription
 
-Metrics-utility is included with Ansible Automation Platform, so no separate installation is needed. The following commands gather the relevant data and generate a CCSP report containing your usage metrics. You can configure these commands as cronjobs to ensure they run at the beginning of every month. See link:https://www.redhat.com/sysadmin/linux-cron-command[How to schedule jobs using the Linux ‘Cron’ utility] for more on configuring using the cron syntax. 
 
+Metrics-utility is included with {PlatformNameShort}, so no separate installation is needed. The following commands gather the relevant data and generate a CCSP report containing your usage metrics. You can configure these commands as cronjobs to ensure they run at the beginning of every month. See link:https://www.redhat.com/sysadmin/linux-cron-command[How to schedule jobs using the Linux ‘Cron’ utility] for more on configuring using the cron syntax. 
```

Comment:
Is CCSP a commonly known acronym? If not, spell it out in the first occurrence.

Reply:
Yup, that's what it is. I updated with a link.